### DISCUSSION TITLE
New README images

### DISCUSSION THREAD

**Discussion by TheRepHelpee** [Maplespe/DWMBlurGlass#281] [Ideas]
### Before I start, here's something to say
Hi, ArbestRi here. I deleted the ArbestRiAgain account and now as I regret it I signed into this account I made in 2021 but never used. 
### Now on to the part I want to discuss.
The images made by Maplespe in README are outdated and use 2.0.1, especially the Aero shader demo:
![Screenshot 2024-04-24 at 7 25 30 PM](https://github.com/Maplespe/DWMBlurGlass/assets/87401373/75f179a7-20b8-42d7-b22a-1787012d747f)
The Windows 7 Aero shader has gotten more accurate since 2.0.1, and I think I'll update the screenshot and possibly other screenshots to showcase 2.1.1 instead if that's okay.

**Comment by Randomuser8219** [comment]
This my new backup account (I'm TheRepHelpee), but here is a concept for a new Windows 10 demo image I made
![Windows 10 x64-2024-04-28-11-15-08](https://github.com/Maplespe/DWMBlurGlass/assets/168323856/164e7651-66af-4b05-accc-f5f8b8d6e2ac)

**Comment by Randomuser8219** [comment]
I completed the rest:
Windows 11 demo:
![Screenshot 2024-04-28 112508](https://github.com/Maplespe/DWMBlurGlass/assets/168323856/3567fe60-0c69-4ead-ba79-b02e22eac66b)
**Blur:**
![Screenshot 2024-04-28 112714](https://github.com/Maplespe/DWMBlurGlass/assets/168323856/f9fd9e57-4551-4eda-9640-8d3f21ded5a1)
**Aero (default balance settings, active and inactive:**
![Screenshot 2024-04-28 112736](https://github.com/Maplespe/DWMBlurGlass/assets/168323856/255ab774-0af2-44d6-87e0-5ff644787ff0)
![Screenshot 2024-04-28 112745](https://github.com/Maplespe/DWMBlurGlass/assets/168323856/28951bc8-59eb-4112-b56d-80f6b3b2e202)
**Acrylic (SystemBackdrop)**
![Screenshot 2024-04-28 112858](https://github.com/Maplespe/DWMBlurGlass/assets/168323856/9b392685-5d61-4f2f-9fbb-f46e8700a817)
**Mica (SystemBackdrop)**
![Screenshot 2024-04-28 112940](https://github.com/Maplespe/DWMBlurGlass/assets/168323856/a442ce58-f841-4e5e-a588-1a60f9ea1156)

**Comment by TheInsane101** [comment]
I love them! However, now that 2.2 is out, you're gonna need to update them again (unless I beat you to it)! XD

**Reply by Randomuser8219** [reply]
I made the screenshots using 2.2.0

**Reply by TheInsane101** [reply]
Oh lmao!
